### PR TITLE
Remdesivir updates

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

These updates were sent to me by @Jeff-Field. I believe I have successfully used to git to give him authorship credit!

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.treatments.md [hunk 1]
```diff
@@ -90,9 +90,9 @@ Therefore, randomized controlled trials are still required.
 ###### Remdesivir
 
 Remdesivir (GS-5734) was developed by Gilead Sciences to treat Ebola Virus Disease.
-It does not have any FDA-approved use.
+At the time of the COVID-19 pandemic, it did not have any have any FDA-approved use.
```
Comment (rando2) [inline content/20.treatments.md:93]:
```suggestion
At the outset of the COVID-19 pandemic, it did not have any have any FDA-approved use.
```

File: content/20.treatments.md [hunk 2]
```diff
@@ -116,17 +116,34 @@ The study included many sites, potentially with variable inclusion criteria and
...
+Therefore, even though the study reported clinical improvement in 68% of the 53 patients ultimately evaluated, due to the significant issues with study design, it could not be determined whether treatment with remdesivir had an effect or whether these patients would have recovered regardless of treatment.
+Another study comparing 5 and 10 day treatment regimens had similar results but was also limited because of the lack of a placebo control [@doi:10.1056/NEJMoa2015301].
+The study did not alter our understanding of the efficacy of remdesivir in treating COVID-19, but the encouraging results motivated placbo controlled studies.
+
+Remdesivir was later tested in a double-blind placebo-controlled phase 3 clinical trial performed at 60 trial sites, 45 of which were in the United states.
+The trial recruited 1063 patients and randomly assigned them to placebo treatment or treatment with remdesivir.
+The treatment was 200 mg on day 1, followed by 100mg on days 2 through 10.
```
Comment (rando2) [inline content/20.treatments.md:125]:
```suggestion
The treatment was 200 mg on day 1, followed by 100 mg on days 2 through 10.
```

File: content/20.treatments.md [hunk 3]
```diff
@@ -116,17 +116,34 @@ The study included many sites, potentially with variable inclusion criteria and
...
-Remdesivir recently entered controlled clinical trials, and as of March 2020, there are six clinical trials underway to treat COVID-19 patients at both early and late stages of infection and in combinations with other drugs [@doi:10.1038/s41422-020-0282-0; @clinicaltrials:NCT04292730; @clinicaltrials:NCT04292899; @clinicaltrials:NCT04280705; @url:https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-000936-23/FR; @clinicaltrials:NCT04252664; @clinicaltrials:NCT04257656].
+Therefore, even though the study reported clinical improvement in 68% of the 53 patients ultimately evaluated, due to the significant issues with study design, it could not be determined whether treatment with remdesivir had an effect or whether these patients would have recovered regardless of treatment.
+Another study comparing 5 and 10 day treatment regimens had similar results but was also limited because of the lack of a placebo control [@doi:10.1056/NEJMoa2015301].
+The study did not alter our understanding of the efficacy of remdesivir in treating COVID-19, but the encouraging results motivated placbo controlled studies.
```
Comment (rando2) [inline content/20.treatments.md:121]:
```suggestion
The study did not alter our understanding of the efficacy of remdesivir in treating COVID-19, but the encouraging results motivated placebo controlled studies.
```

**Comment by rando2** [comment]
@aadattoli, did you want to read through these updates before I merge them in? I remember you also worked on this section!

**Comment by aadattoli** [comment]
Hey @rando2 , done.

Thanks for keeping me posted!

Best

**Pull request merged**